server/svix-server/src/queue/redis.rs

```diff
@@ -40,46 +48,56 @@ use super::{
 
 // FIXME: Change unwraps to have our own error type for the queue module entirely
 
+/// This is the key of the main queue. As a KV store, redis places the entire stream under this key.
+/// Confusingly, each message in the queue may have any number of KV pairs.
 const MAIN: &str = "svix_{queue}_v3_main";
 
-// TODO: Stream based delayed queue
-// const DELAYED: &str = "svix_queue_v3_delayed";
+// TODO: Migrate DELAYED queue to use streams too
+
+/// The key for the DELAYED queue in which scheduled messages are placed. This is the same DELAYED
+/// queue as v2 of the queue implementation.
+const DELAYED: &str = "{queue}_svix_delayed";
 
-// FIXME: hash-tags here enable clustering but prevent proper sharding of lists
+// v2 KEY CONSTANTS
 const LEGACY_MAIN: &str = "{queue}_svix_main";
 const LEGACY_PROCESSING: &str = "{queue}_svix_processing";
-// TODO: Rename LEGACY_DELAYED
-const DELAYED: &str = "{queue}_svix_delayed";
 
+// v1 KEY CONSTANTS
```

Comment:
Maybe instead of LEGACY_LEGACY just call it LEGACY_V1_MAIN and etc? And the other one LEGACY_V2?
Much better than LEGACY_LEGACY and documenting it in comments, no?